star-rx72n-firmware/lib/rx_spi_comm/src/rx_spi_comm.c

```
@@ -534,6 +534,82 @@ rx_err_t rx_spi_comm_send_nack(rx_spi_comm_handle_t* handle, const uint16_t sequ
+  uint32_t wire_len = 0;
+
+  err = rx_frame_encode(&handle->encoder, &reset_ack_frame, wire_buffer, &wire_len);
+  if (err != k_rx_ok) {
+    return err;
+  }
+
+  return internal_spi_transfer(handle, wire_buffer, wire_len, NULL, 0);
+}
+
+rx_err_t rx_spi_comm_set_control_callbacks(rx_spi_comm_handle_t*    handle,
+                                            rx_spi_comm_control_cb_t on_ping,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_spi_comm_set_control_callbacks' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)